### DISCUSSION TITLE
Models do not show up in ControlNet

### DISCUSSION THREAD

**Discussion by funnymataleao** [omniinfer/sd-webui-cloud-inference#42] [Q&A]
Hi. 
Thank you for your service, I like it very much. Generations are very fast and it is cheap. I configured everything according to the instructions, bound my key, and enabled Cloud Interface, but unfortunately, there are no models in ControlNet. What am I doing wrong?

**Comment by AnyISalIn** [comment]
Could you please provide the ControlNet version and the WebUI version?

**Reply by funnymataleao** [reply]
Hi. There is no news? I still can't use ControlNet :(

**Reply by AnyISalIn** [reply]
its works for me. have you click refresh button 
<img width="779" alt="image" src="https://github.com/omniinfer/sd-webui-cloud-inference/assets/16937838/688dc2d0-b953-41e0-9d58-e3a0bc4eced0">

**Reply by funnymataleao** [reply]
Of course I tried that. The ControlNet models still don't appear in the drop-down list, it's just empty.

**Reply by funnymataleao** [reply]
Hi. There is no news?

**Comment by funnymataleao** [comment]
Hey. For sure. Sorry for the long reply. WebUI v1.6.0 and ControlNet v1.1.410
